pkg/tasks/tasks.go

```diff
@@ -104,7 +104,15 @@ func WithHostnameOSAndProbes(t Tasks) Tasks {
 // WithFullInstall with install binaries (using WithBinariesOnly) and
 // orchestrate complete cluster init
 func WithFullInstall(t Tasks) Tasks {
-	return WithBinariesOnly(t).
+	return Tasks{
+		{
+			Fn: func(s *state.State) error {
+				s.Logger.Infoln("Disable nm-cloud-setup nodes...")
```

Comment:
Let’s move this logger to the executor function (disableNMCloudSetup) and only show it if we are about to execute the command. Also, I’d include some additional details such as: “Disabling nm-cloud-setup… The node will be rebooted…”.